key.core/src/main/java/de/uka/ilkd/key/speclang/njml/Translator.java

```diff
@@ -1031,6 +1033,21 @@ public Object visitPrimarySuffixCall(JmlParser.PrimarySuffixCallContext ctx) {
                 raiseError(ctx, e);
             }
         }
+
+        // support JML built-ins
+        if (fullyQualifiedName.startsWith("\\")) {
+            try {
+                return processJmlBuiltInFunction(fullyQualifiedName, accept(ctx.expressionlist()));
+            } catch (Exception e) {
+                // weigl: only log error. There is no reason why a method should not start with a \\
+                // in JML.
+                // The next version will have, e.g., \seq and \map classes.
+                addWarning(ctx,
+                    "%s was not understand as a JML built-in function. I assume it to be a method or class."
```

Comment:
language: "%s is not a built-in JML function name. I assume it is a method or class name."

Reply:
:question: